portfolio/src/main/webapp/index.html

```diff
@@ -18,13 +18,22 @@ <h1>Kiara's Portfolio</h1>
           I was born, partially raised and living in New Orleans. After hurricane Katrina, I moved to Mississippi
           by my grandfather side of the family. I lived in Mississippi until I was in the 7th grade then moved back 
           to New Orleans for the rest of my school years and college.</p> 
+      
+      <p> So far this is my favorite picture of myself. </p>
+      <img id="subcontent" src="Images/fav-pic.JPG">
```

Comment:
id should be unique per html document. If you want to associate a symbol to multiple elements, either for styling or to find them in js, then class is a better fit for that use case.